lib/capistrano/tasks/deploy.cap

```diff
@@ -119,38 +119,39 @@ namespace :deploy do
          "and add release notes"
   end
 
-  after :cleanup, :deploy_pt_stories do
-    pt_config_file = File.expand_path("../../../../config/pivotaltracker.yml", __FILE__)
-    next if fetch(:stage) != :staging || !File.exists?(pt_config_file)
-
-    require 'pivotal-tracker'
-    tracker = YAML.load_file pt_config_file
-    %i[token project_id].each do |n|
-      raise "#{n} not set in pivotaltracker.yml" unless tracker[n]
-    end
-
-    PivotalTracker::Client.token = tracker[:token]
-    PivotalTracker::Client.use_ssl = true
-
-    unpakt_project = PivotalTracker::Project.find(tracker[:project_id])
-    stories =
-      unpakt_project.stories.all(:state => "finished", :story_type => ['bug', 'feature'])
-
-    staging_deploy_tag = `git tag | grep staging | tail -n1`
-
-    stories.each do |story|
-      next if story.labels&.include? "not merged"
-      puts "Searching for #{story.id} in local git repo."
-      search_result = `git log --grep #{story.id} #{staging_deploy_tag}`
-      if search_result.length > 0
-        puts "Found #{story.id}, marking as delivered."
-        story.notes.create(:text => "Delivered by staging deploy script.")
-        story.update({"current_state" => "delivered"})
-      else
-        puts "Could not find #{story.id} in git repo."
-      end
-    end
-  end
+  # after :cleanup, :deploy_pt_stories do
+  #   pt_config_file = File.expand_path("../../../../config/pivotaltracker.yml", __FILE__)
+  #   next if fetch(:stage) != :staging || !File.exists?(pt_config_file)
+  #
+  #   require 'pivotal-tracker'
```

Comment:
you could replace `pivotal-tracker` with `tracker-api`
https://github.com/dashofcode/tracker_api